### ISSUE TITLE
dependency problem

### ISSUE THREAD

**Issue by antonkulaga** [drdozer/shortbol#1]
uk.co.turingatemyhamster#datatree-core_sjs0.6_2.11;develop-0.2.1: not found

**Comment by drdozer** [comment]
Hey -- thanks for flagging this. The datatree project needs a thorough
clean and a public release.

On 21 June 2016 at 13:27, Anton Kulaga notifications@github.com wrote:

> uk.co.turingatemyhamster#datatree-core_sjs0.6_2.11;develop-0.2.1: not found
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/drdozer/shortbol/issues/1, or mute the thread
> https://github.com/notifications/unsubscribe/AAjnXLP6eVR4CetWE1UYpscTK304RAPrks5qN9ingaJpZM4I6rSS
> .

## 

Dr Matthew Pocock
Turing ate my hamster LTD
mailto: turingatemyhamster@gmail.com

Integrative Bioinformatics Group, School of Computing Science, Newcastle
University
mailto: matthew.pocock@ncl.ac.uk

gchat: turingatemyhamster@gmail.com
msn: matthew_pocock@yahoo.co.uk
irc.freenode.net: drdozer
skype: matthew.pocock
tel: (0191) 2566550
mob: +447535664143

**Comment by drdozer** [comment]
Actually, thinking about it, the current build has no code that depends on
this, so you should be able to comment it out

On 21 June 2016 at 13:40, Matthew Pocock turingatemyhamster@gmail.com
wrote:

> Hey -- thanks for flagging this. The datatree project needs a thorough
> clean and a public release.
> 
> On 21 June 2016 at 13:27, Anton Kulaga notifications@github.com wrote:
> 
> > uk.co.turingatemyhamster#datatree-core_sjs0.6_2.11;develop-0.2.1: not
> > found
> > 
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > https://github.com/drdozer/shortbol/issues/1, or mute the thread
> > https://github.com/notifications/unsubscribe/AAjnXLP6eVR4CetWE1UYpscTK304RAPrks5qN9ingaJpZM4I6rSS
> > .
> 
> ## 
> 
> Dr Matthew Pocock
> Turing ate my hamster LTD
> mailto: turingatemyhamster@gmail.com
> 
> Integrative Bioinformatics Group, School of Computing Science, Newcastle
> University
> mailto: matthew.pocock@ncl.ac.uk
> 
> gchat: turingatemyhamster@gmail.com
> msn: matthew_pocock@yahoo.co.uk
> irc.freenode.net: drdozer
> skype: matthew.pocock
> tel: (0191) 2566550
> mob: +447535664143

## 

Dr Matthew Pocock
Turing ate my hamster LTD
mailto: turingatemyhamster@gmail.com

Integrative Bioinformatics Group, School of Computing Science, Newcastle
University
mailto: matthew.pocock@ncl.ac.uk

gchat: turingatemyhamster@gmail.com
msn: matthew_pocock@yahoo.co.uk
irc.freenode.net: drdozer
skype: matthew.pocock
tel: (0191) 2566550
mob: +447535664143

**Comment by antonkulaga** [comment]
Ok. I wonder, is it going to be scala/scalajs version of SBOL? And why is it mixed with python server?

**Comment by drdozer** [comment]
It is a shorthand for nested RDF- like tree expressions. The driving use case is sbol. But I will also be adding modes for owl and prov-o and some other terminologies and models

**Comment by drdozer** [comment]
The current build depends on this again. I'm going to cut a release of datatree and then update the dependency here to match.

**Comment by drdozer** [comment]
Right, I've pushed an updated datatree that now correctly cross-compiles and updated the dependency here. However, it isn't building because it depends on my graphviz library which also is not being correctly published. Oh the joys.
